src/main/java/uk/gov/hmcts/cmc/claimstore/models/offers/Settlement.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.cmc.claimstore.models.offers;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import uk.gov.hmcts.cmc.claimstore.exceptions.IllegalSettlementStatementException;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Settlement {
+
+    private List<PartyStatement> partyStatements = new ArrayList<>();
+
+    public void makeOffer(Offer offer, MadeBy party) {
+        assertOfferCanBeMadeBy(party);
+        partyStatements.add(new PartyStatement(StatementType.OFFER, party, offer));
+    }
```

Comment:
Perhaps more flexible `public void addPartyStatement(...)` would be better?

Reply:
I disagree.

Reply:
Future stories will provide `acceptOffer` and `rejectOffer` methods providing a consistent interface instead of a generic method that allows to insert whatever.

Reply:
ok